index.html

```diff
@@ -10,6 +10,10 @@
   rel="stylesheet"
   href="https://fonts.googleapis.com/icon?family=Material+Icons"
 />
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
```

Comment:
It is worth mentioning here, that using fonts from google directly might cause privacy issue as it collects personal data. You can read more about that here:
https://www.cookieyes.com/documentation/google-fonts-and-gdpr/
We can download the fonts and use them directly.

Reply:
i set it here because in mui installation set the font here , but now i remove it